### PR TITLE
update generate changelog workflow

### PR DESCRIPTION
**Pull request by avidit**
### Purpose

update generate changelog workflow

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated
- [ ] This PR contains no files larger than 50 MB

### Release Notes

(FILL ME IN) Brief description of the fix / enhancement. **Mandatory section**

### Reviewers

(FILL ME IN) Reviewer 1 (If possible, assign the Reviewer for the PR)

(FILL ME IN, optional) Any additional notes to reviewers or testers.

### FYIs

(FILL ME IN, Optional) Names of anyone else you wish to be notified of

### CODE REVIEW

File: .github/workflows/generate_changelog.yml [hunk 1]
```diff
@@ -21,12 +21,16 @@ jobs:
...
           myToken: ${{ secrets.GITHUB_TOKEN }}
           head-ref: ${{ github.event.inputs.head-ref }}
           base-ref: ${{ github.event.inputs.base-ref }}
       - name: Summary
         run: |
-          echo "# Changelog" >> $GITHUB_STEP_SUMMARY
-          echo "${{ steps.generate_changelog.outputs.changelog }}" >> $GITHUB_STEP_SUMMARY
+          cat <<EOF >> $GITHUB_STEP_SUMMARY
+          # Changelog
+          [Compare changes](${{ format('{0}/{1}/compare/{2}...{3}', github.server_url, github.repository, github.event.inputs.base-ref, github.event.inputs.head-ref) }})
```
Comment (QilongTang) [inline .github/workflows/generate_changelog.yml:33]: So the changelog is reflected in action result or somewhere else?

Comment (QilongTang) [reply .github/workflows/generate_changelog.yml:33]: Sounds good to me!

**Review by QilongTang — APPROVED** [review]

File: .github/workflows/generate_changelog.yml (hunk 1 above)
Comment (sm6srw) [reply .github/workflows/generate_changelog.yml:33]: 👍

Comment (avidit) [reply .github/workflows/generate_changelog.yml:33]: https://github.com/DynamoDS/Dynamo/actions/runs/8508112969

**Pull request merged**